externals/llvm-external-projects/torch-mlir-dialects/include/torch-mlir-dialects/Dialect/Tcp/IR/TcpOps.td

```diff
@@ -121,6 +139,25 @@ def Tcp_MulOp : Tcp_BinaryElementwiseOp<"mul", [SameOperandsAndResultElementType
   let assemblyFormat = "$in1 `,` $in2 attr-dict `:` type($in1) `,` type($in2) `->` type($out)";
 }
 
+def Tcp_DivOp : Tcp_BinaryElementwiseOp<"div", [SameOperandsAndResultElementType]> {
```

Comment:
What is the behavior on integer division by zero?

Reply:
Should we just make it TBD, just like `arith` (https://mlir.llvm.org/docs/Dialects/ArithOps/#arithdivsi-mlirarithdivsiop) since we are anyways lowering it down to the corresponding arith op.

Reply:
Or just say it is undefined behavior?

Reply:
Thanks. Will update the description to denote division by zero results in UB.

Reply:
For signed division dividing `INT_MIN` by `-1` should also invoke UB.

Reply:
Only supports floating-point division. Integer division support is future work (TBD).